components/sup/src/manager/service/hooks.rs

```diff
@@ -54,7 +54,7 @@ where
 }
 
 #[derive(Debug, Copy, Clone)]
-pub struct ExitCode(i32);
+pub struct ExitCode(pub i32);
```

Comment:
Not quite sure this needs to be public; see comment below on adding functions to this struct instead.